packages/synapse-react-client/src/components/styled/ColumnHeader.tsx

```diff
@@ -0,0 +1,88 @@
+import { HeaderContext } from '@tanstack/react-table'
+import { Box, IconButton, Tooltip } from '@mui/material'
+import { HelpTwoTone } from '@mui/icons-material'
+import IconSvg from '../IconSvg'
+import React from 'react'
+
+type ColumnHeaderProps = {
+  title?: React.ReactNode
+  helpText?: React.ReactNode
+  additionalButtons?: React.ReactNode
+
+  // TODO: Replace with props that can be passed to a reusable filter control
+  filterControl?: React.ReactNode
```

Comment:
Longer term vision would be that the caller passes the possible values/ranges, and statistics if appropriate, then this component would handle the UI.

That's a lot of work, so for now, the caller will just pass the rendered filter UI.